src/state/apis/zapper/zapperApi.ts

```diff
@@ -461,10 +468,21 @@ export const zapper = createApi({
                   const defiType = DefiType.Staking
 
                   const topLevelAsset = (() => {
-                    const maybeLpAsset = asset.tokens.find(
+                    const chainId = zapperNetworkToChainId(asset.network)
+                    if (!chainId) throw new Error('chainIs is required')
+
+                    const maybeTopLevelLpAssetId = toAssetId({
+                      chainId,
+                      assetNamespace:
+                        chainId === bscChainId ? ASSET_NAMESPACE.bep20 : ASSET_NAMESPACE.erc20,
+                      assetReference: asset.address,
+                    })
+                    const maybeTopLevelLpAsset = assets[maybeTopLevelLpAssetId]
+                    if (maybeTopLevelLpAsset?.isPool) return asset
+                    const maybeUnderlyingLpAsset = asset.tokens.find(
```

Comment:
Fixing parsing of *some* Zapper opportunities which were surfacing the wrong top-level asset, hence making routing to swapper borked